app/org/catalyst/slackservice/controllers/AuthController.java

```diff
@@ -73,11 +70,6 @@ public AuthController(AppService service, AppConfig config, MessagesApi messages
                         "error", response.error))));
             }
 
-            var teamName = response.team != null ? response.team.name : null;
```

Comment:
I'm a bit hesitant to remove this entirely, mostly because it was in previous analytics reports and that we won't be able to get these values later on. Did we confirm that there's no need for the team name?